services/hidadapter.md

```diff
@@ -0,0 +1,87 @@
+# HID Adapter Configurator
+
+    identifier: 0x1e5758b5
+    
+A service for configuring how Jacdac device map to HID input events. Users can have multiple configurations and swap between them using the `swapTo` or `next` commands.
+
+## Registers
+
+    rw num_configurations: u8 @ 0x80
+    
+The number of configurations stored on the server.
+
+    rw current_configuration: u8 @ 0x81
+
+The current configuration the server is using.
+
+## Commands
+
+    command get_configuration @ 0x80 {
+        results: pipe
+        configuration_number: u8
+    }
+    pipe report configuration {
+        configuration_number: u8
+        binding_index: u8
+        device_id: u8[8]
```

Comment:
we typically use `u64` (which requires 32 bit alignment) for device id

Reply:
Updated